lib/mix/lib/mix/task.ex

```diff
@@ -48,13 +48,20 @@ defmodule Mix.Task do
   @doc false
   defmacro __using__(_opts) do
     quote do
-      Enum.each [:shortdoc, :recursive, :preferred_cli_env],
+      Enum.each Mix.Task.supported_attributes,
```

Comment:
Same as above IMO, but if we don't want to change both lines here, we should at least use parens in the call to `Mix.Task.supported_attributes/0`.